.github/workflows/tests.yml

```diff
@@ -374,7 +374,7 @@ jobs:
         working-directory: complement/dockerfiles
 
       # Run Complement
-      - run: go test -v -tags synapse_blacklist,msc2403,msc2946,msc3083 ./tests/...
```

Comment:
There's duplication here with `complement.sh`. (The latter of which now has a merge conflict, I think).

Is there some way to make both of these places read the excluded tags from a single file?

Possibly out-of-scope for this change.

Reply:
It is really annoying, see matrix-org/complement#185. I don't have good ideas how to solve it, but I think it is out of scope of this PR.

Reply:
Glad to see we're both annoyed by this! Agreed, let's leave it.